### PR TITLE
Add manipulation function to pointer example

### PR DESCRIPTION
**Pull request by mmichaelb**
Implements the feature from #21

### CODE REVIEW

File: src/pointer.go [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 
-import (  
-    "fmt"
+import (
+	"fmt"
 )
 
-func main() {  
-    b := 255
-    var a *int = &b                       //Saving b's address into variable a
```
Comment (aprabhat) [inline src/pointer.go:9]: Thank you @mmichaelb  for the valuable contribution.
I think its better so start with basic things first like how to  declare a pointer, what is the significance of '*' and '&' sign. So please add some code comments related to that.

File: src/pointer.go [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+	b := 255
+	var a *int = &b //Saving b's address into variable a
+	fmt.Printf("Type of a is %T\n", a)
 	fmt.Println("address of b is", a)
-	
 
-   // Creating pointers using the new function
+	// Creating pointers using the new function
 
-   size := new(int)  // we can imagine that "new" keyword is a pointer
```
Comment (aprabhat) [inline src/pointer.go:16]: Please add correct definition of  **new** as its a built in function that allocates memory and return address of it.

**Review by aprabhat — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please see the review comments.

File: src/pointer.go (hunk 1 above)
Comment (mmichaelb) [reply src/pointer.go:9]: I definitely agree with you on this one. However, my pull request/referred issue was really only about the implementation of the pointer function to show another Pointer related feature. Anyway, I am happy to deal with this matter as part of another issue.

File: src/pointer.go (hunk 2 above)
Comment (mmichaelb) [reply src/pointer.go:16]: As already mentioned, the same applies for this one. I totally agree with you but this pull request/issue simply does not address this topic/problem. In order to maintain the issue structure, I would rather deal with this problem in another issue/pull request.

**Review by rajkumarGosavi — CHANGES_REQUESTED** [review]
Needs a lot of comments as readers usually find this topic difficult to learn it would be better to help them as much as you can. Dig deeper in this topic and it will also help you in your life.

File: src/pointer.go (hunk 2 above)
Comment (aprabhat) [reply src/pointer.go:16]: >  In order to maintain the issue structure, I would rather deal with this problem in another issue/pull request.

I really appreciate that! Thank you. Merging this request and will open new issues for improvement of code comments.

**Comment by sagar-jadhav** [comment]
@mmichaelb I appreciate your contribution 👍 

#73 & #83 is created for improvement related to this PR. Can you have a look into those issues?

**Comment by mmichaelb** [comment]
Sure, of course! Right now I'm pretty busy at work, but I'll see that I can find a few minutes 😉

**Comment by sagar-jadhav** [comment]
@mmichaelb Take Your Time :)

**Pull request merged**